spec/_superseded_operations.yaml

```diff
@@ -586,7 +586,7 @@ $schema: ./json_schemas/_superseded_operations.schema.yaml
   operations:
     - POST
 /_opendistro/_sql/settings:
-  superseded_by: /_plugins/_sql/settings
+  superseded_by: /_plugins/_query/settings
```

Comment:
Does this mean `/_plugins/_sql/settings` actually doesn't exist?

Reply:
Yeap! 
<img width="663" alt="image" src="https://github.com/user-attachments/assets/3d55af4d-eca5-48c9-b66b-2cf19f857361">
<img width="660" alt="image" src="https://github.com/user-attachments/assets/c03bc209-e236-4dca-a57d-cbb0a4d721bb">